### DISCUSSION TITLE
A question regarding embedded images.

### DISCUSSION THREAD

**Discussion by BG-Cypo** [Tyrrrz/DiscordChatExporter#723] [Q&A] [answered]
Let's say a scenario. I successfully exported an entire channel, and it has images. If the channel ever get deleted, do the images stay?

**Comment by Yudi** [comment, marked as answer]
If you have activated the "Download media" option, they'll stay, otherwise they'll be gone.

**Reply by BG-Cypo** [reply]
Thank you!

**Reply by scotofil** [reply]
Apologies for the necro!
I'm new to the app; I've installed v2.39.1 just now on Windows, and did my first exports using the GUI, however neither among the app-level settings, nor among the export-level settings did I find "Download media". @Yudi Is this feature still available in 2.39.1?

**Reply by Yudi** [reply]
@scotofil I believe the option is now named "Download assets" and it's available under the export settings dialog

**Reply by scotofil** [reply]
Thanks a lot! Turns out I went full retard and didn't recognise that in the export settings dialog there's a scroll bar and some additional parameters apart from what's visible in the initial state of the expanded setting window. My bad, and thanks a lot for the prompt and patient response!

**Reply by Yudi** [reply]
No problem, glad to help! :-)
